server/lib/budget.js

```diff
@@ -134,6 +134,7 @@ export async function getTotalMoneyManagedAmount(host, { startDate, endDate, cur
 
   const results = await sumCollectivesTransactions(ids, {
     startDate,
+    excludeRefunds: false,
```

Comment:
This was needed to fix the following tests:

```es6
      expect(await collective.getBalance()).to.eq(0);
      expect(await host.getBalance()).to.eq(-1200); // Will be -200 after settlement (platform tip)
      expect(await host.getTotalMoneyManaged()).to.eq(-1200);
```

Without that, `host.getTotalMoneyManaged()` returned `-1000`